app/ads/components/ui/badge.tsx

```diff
@@ -3,48 +3,48 @@ import { Badge } from "@/components/ui/badge"
 import { cn } from "@/lib/utils"
 
 interface CustomBadgeProps extends React.HTMLAttributes<HTMLDivElement> {
-    className?: string
+  className?: string
 }
 
 export function ActiveBadge({ className, ...props }: CustomBadgeProps) {
-    return (
-        <Badge
-            variant="active"
-            className={cn("bg-success-light text-success hover:bg-success-light border-transparent", className)}
-            {...props}
-        />
-    )
+  return (
```

Comment:
what's the difference between this component and the one from ui? Do we need to create another component for calling `Badge`?